heroic-core/src/main/java/com/spotify/heroic/metric/MetricManagerModule.java

```diff
@@ -88,6 +89,11 @@
      */
     private final boolean failOnLimits;
 
+    /**
+     * Limit which queries that are logged
+     */
+    private final OptionalLimit logQueriesThresholdDataPoints;
```

Comment:
This variable name reads a bit weird, either `DataPoints` should come before `Threshold`, or use something simpler like `logQueryThreshold`.